src/Workspaces/CSharp/Portable/Rename/LocalConflictVisitor.cs

```diff
@@ -38,11 +38,16 @@ public override void VisitMethodDeclaration(MethodDeclarationSyntax node)
 
         public override void VisitBlock(BlockSyntax node)
         {
+            VisitBlockStatements(node, node.Statements);
```

Comment:
So we have 2 additional allocations per keystroke while Inline Rename is active (which accounts for a tiny percentage of overall keystrokes). I don't see a big problem here. Does this sound okay @rchande?